recipe/activate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Instead of including the activation script here, should we add these changes to `cuda-nvcc`'s activation script (and just keep the `nvcc.profile.patch` change here)?

Reply:
Then the activation script which defines CUDA_LINK_DIRS and the patch which uses CUDA_LINK_DIRS would be in different feedstocks. That seems confusing.

Reply:
Adding this activation script here should be fine, although I think it needs to go to the `cuda-nvcc-impl` package to follow the original intent: https://github.com/conda-forge/cuda-nvcc-feedstock/issues/12#issuecomment-1533562469

Reply:
I put the activation script with cuda-nvcc-tools because that is the package which contains nvcc.profile. nvcc.profile is the only file that needs the activation script, so it would seem odd to separate the two.